modules/specialized-guides/pages/large-deployments/hub-install.adoc

```diff
@@ -1,35 +1,75 @@
 [[lsd-hub-install]]
-= Hub XMLRPC API Deployment
+= Hub Deployment with XMLRPC
 
-Before you begin, you need to install the ``hub-xmlrpc-api`` package, and configure the Hub Server to use the API.
+For a hub environment, first deploy {productname}.
+Add [option]``--hubxmlrpc-replicas 1`` to the [command]``mgradm install`` command line.
+For example:
 
+----
+mgradm install podman suma.example.com --hubxmlrpc-replicas 1
+----
+
+For more information about deploying with [command]``mgradm``, see xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc[].
+
+Then prepare SSL certificates for the peripheral servers.
+For more information about SSL certificates, see xref:specialized-guides:large-deployments/hub-reqs.adoc#lsd-hub-reqs-certs[].
+
+
+.Procedure: Using Third Party Certificates
+. Preliminary Requirement: Hub server installation
```

Comment:
I think the best workflow here is to prepare 3rd party certificates for both Hub and Peripheral servers first and then deploy them, to ensure that the hub is using the same CA.

Hub:
mgradm install podman --ssl-ca-root CA-Certificate.crt --ssl-server-cert hub.crt --ssl-server-key hub.key --hubxmlrpc-replicas 1

Peripheral servers:
mgradm install podman --ssl-ca-root CA-Certificate.crt --ssl-server-cert server.crt --ssl-server-key server.key

Reply:
I tried to separate deployment with 3rd party and self-generated certs. They are now standalone procedure.  I hope the basics are ok, now, but I wasn't completely sure with all the details.  I want reviews again ;)